about.html

```diff
@@ -12,53 +12,89 @@
 </head>
 
 <body>
+
     <div class="container about-page">
-     
-    About
 
+    <header class="tophead">
+      <a href="index.html">
+        <img src="img/lambda-black.png" alt="site-logo">
```

Comment:
```suggestion
        <img src="img/lambda-black.png" alt="site-logo">
```
If you add title="yourTitleHere" as a property to your img tags it will improve you SEO strength.